snap/local/localization_launcher.sh

```diff
@@ -29,4 +29,4 @@ else
 fi
 
 ros2 launch nav2_bringup localization_launch.py \
-  params_file:=${CONFIG_FILE} map:=${MAP_FILE} use_sim_time:=${SIMULATION}
+  params_file:=${CONFIG_FILE} map:=${MAP_FILE}
```

Comment:
why the use_sim_time / simulation option has been removed?

Reply:
Yes actually we can keep the option on localization and slam launchers, but not navigation (ie, [test job fails](https://github.com/ubuntu-robotics/ros2-nav2-snap/actions/runs/10597697531/job/29369103804) if so).

Reply:
the navigation_launcher has the use_sim_time parameter tho https://github.com/ros-navigation/navigation2/blob/jazzy/nav2_bringup/launch/navigation_launch.py#L34 , maybe it is just the amcl use_sim_time parameter in the template configuration that has to be removed?